common/lib/xmodule/xmodule/x_module.py

```diff
@@ -361,6 +362,14 @@ def xblock_kvs(self):
         self.save()
         return self._field_data._kvs  # pylint: disable=protected-access
 
+    @lazy
+    def _unwrapped_field_data(self):
+        """
+        This property hold the value _field_data here before we wrap it in
+        the LmsFieldData or OverrideFieldData classes.
+        """
+        return self._field_data
```

Comment:
It looks like there are a couple other places in edx-platform that set `_field_data` (outside of `bind_for_student`). Those places should probably reset `_unwrapped_field_data` (or trigger something that resets it automatically)